tools/openshift-ci/Dockerfile.centos

```diff
@@ -10,15 +10,18 @@ ADD . /verification-tests/
 RUN set -x && \
     SCL_BASE_PKGS="centos-release-scl scl-utils-build" && \
     INSTALL_PKGS="rh-ruby26 rh-ruby26-ruby-devel rh-git218 bsdtar" && \
+    JQ_PKG="jq" && \
+    curl -L https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm -o /tmp/epel-release-latest-7.noarch.rpm && \
+    yum install -y /tmp/epel-release-latest-7.noarch.rpm && \
+    yum install -y $JQ_PKG && \
```

Comment:
could you integrate this in same command we download `SCL_BASE_PKGS`? Running `yum` twice is taking a lot of time. I would say, download all `SCL_BASE_PKGS`, `JQ_PKG` and `INSTALL_PKGS` with one `yum` command. Also please use option `--setopt=skip_missing_names_on_install=False`, otherwise we need a separate `rpm -V` line which is another needless slowdown. Better fail early.